Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -330,13 +330,47 @@ public final class RewardAddOnSelectionViewModel: RewardAddOnSelectionViewModelT
       )
     }
 
-    Signal.zip(project, baseReward, refTag)
+    let calculatedPledgeTotal = Signal.combineLatest(
+      allRewardsShippingTotal,
+      allRewardsTotal
+    )
+    .map { shippingCost, rewardBaseAmount -> Double in
```

Comment:
You should be using the methods Afees refactored, specifically for this reason, in the last PR:
`.map(calculatePledgeTotal)`

Reply:
Also I don't think this should be in an `observeValues` block.